src/main/java/oracle/kubernetes/operator/Main.java

```diff
@@ -979,6 +983,113 @@ else if (StartupControlConstants.AUTO_STARTUPCONTROL.equals(sc)) {
     return env;
   }
   
+  private static Step checkServerServices(DomainPresenceInfo info, Step next) {
+    Collection<String> allServers = new ArrayList<>();
+    Collection<ServerStartupInfo> ssic = new ArrayList<>();
+    
+    WlsDomainConfig scan = info.getScan();
+
+    // Iterate all servers
+    for (WlsClusterConfig wlsClusterConfig : scan.getClusterConfigs().values()) {
+      for (WlsServerConfig wlsServerConfig : wlsClusterConfig.getServerConfigs()) {
+        String serverName = wlsServerConfig.getListenAddress();
+        if (!allServers.contains(serverName)) {
+          allServers.add(serverName);
+          ssic.add(new ServerStartupInfo(wlsServerConfig, wlsClusterConfig, null, null));
+        }
+      }
+    }
+    for (Map.Entry<String, WlsServerConfig> wlsServerConfig : scan.getServerConfigs().entrySet()) {
```

Comment:
It seems to me that the ServerConfigs here may include admin server too. If so, do we need to exclude it in the "ssic" list that we generate? The "ssic" list is later used in ManagedServerServicesStep, which I assume should not impact admin server. Or maybe the admin server is excluded later somewhere?

Reply:
Yes, this includes admin server here so that we validate that the admin server specific Service is created and correct.

This list of ServerStartupInfo ("ssic") is different than the other list.  I used similar data because we need details about port/nodePort to correctly create the services.